uiTests/build.gradle

```diff
@@ -20,8 +18,9 @@ task syncTestResults(type: Sync) {
 }
 
 test {
-  // Exclude UI tests when running `./gradlew test`, unless a `-PenableUiTests` Gradle flag is provided.
```

Comment:
Simplified a bit, a separate `-PenableUiTests` project property is no longer needed